_docs/style/focus-states.md

```diff
@@ -364,7 +364,7 @@ The beacon comes with a glow baked in, so when it receives focus, the existing g
 
 ### Focusable containers
 
-Some elements, such as [Choice Tiles]() feature a clickable container. In these cases, focus is applied to the entire container, according to normal focus rules.
+Some elements, such as [Option Tiles]() feature a clickable container. In these cases, focus is applied to the entire container, according to normal focus rules.
```

Comment:
```suggestion
Some elements, such as [Option Tiles]({{site.baseurl}}/components/option-tiles.html) feature a clickable container. In these cases, focus is applied to the entire container, according to normal focus rules.
```

Reply:
The focusable containers section is no longer available. I'm not sure why you saw this?